tests/test_profiles.py

```diff
@@ -507,3 +505,28 @@ def test_kinetic_pressure(self):
         assert np.all(data2["Te_r"] == data2["Ti_r"])
         np.testing.assert_allclose(data1["p"], data2["p"])
         np.testing.assert_allclose(data1["p_r"], data2["p_r"])
+
+    @pytest.mark.unit
+    def test_line_average(self):
+        """Test that line-averaged profiles are correct."""
+        ne0 = 3e19
+        Te0 = 2e3
+        ne = PowerSeriesProfile(2 * ne0 * np.array([1, -1]), modes=[0, 1])
+        Te = PowerSeriesProfile(2 * Te0 * np.array([1, -1]), modes=[0, 1])
+
+        with pytest.raises(UserWarning):
```

Comment:
what warning does this raise?

Reply:
The profiles are not even power series. I made them linear so the average was simpler